src/ngraph/op/fused/batch_mat_mul_transpose.cpp

```diff
@@ -31,13 +34,67 @@ op::BatchMatMulTranspose::BatchMatMulTranspose(const Output<Node>& arg0,
                                                const Output<Node>& arg1,
                                                bool transpose_arg0,
                                                bool transpose_arg1)
-    : Op({arg0, arg1})
+    : FusedOp({arg0, arg1})
     , m_transpose_arg0(transpose_arg0)
     , m_transpose_arg1(transpose_arg1)
 {
     constructor_validate_and_infer_types();
 }
 
+NodeVector op::BatchMatMulTranspose::decompose_op() const
+{
+    const PartialShape& arg0_pshape = get_input_partial_shape(0);
+    if (arg0_pshape.is_dynamic())
+    {
+        throw ngraph_error("Arg0 needs to have static shape to decompose");
```

Comment:
Use `NODE_VALIDATION_CHECK` instead of throwing here and below. It will produce a more useful error message.

Reply:
done